zeppelin-web/add_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+( sudo apt-get install libfontconfig -y || sudo yum install fontconfig -y ) || exit 0;
```

Comment:
@Leemoonsoo Mmm, the main goal of this PR is give possibility run 
```
mvn clean package -Pbuild-distr
```
(see http://zeppelin.apache.org/docs/snapshot/install/build.html#package)
without failures. As you can see in my PR (screenshot section) zeppelin-web builds about 40 minutes. So if hidden dependency was not installed automatically build will be failed after about 50 minutes. And only after this i could show this message. I think this is vesting of time and money (if you use EC2). More over in this case user should waiting for this a front of monitor...
This is normal situation when linux package already installed and user try install it again - this will not changed system state.
BTW, zeppelin developed not only at Ubuntu nodes. What should we do if this is Redhat or Centos or OsX? Fortunately, Windows phantomjs package do not need these hidden dependencies, but Lunuxes need. I suspect that OsX also...